pkgs/development/libraries/libaom/default.nix

```diff
@@ -1,5 +1,5 @@
 { lib, stdenv, fetchzip, yasm, perl, cmake, pkg-config, python3
-, enableButteraugli ? true, libjxl
+, libjxl
```

Comment:
libjxl is as far as I know only used for butteraugli, (and is as of now unused in the actual package). So `libjxl` should be removed as well